packages/webapp/src/pages/Environment/Settings/ConnectUISettings/index.tsx

```diff
@@ -142,154 +189,57 @@ export const ConnectUISettings = () => {
                             )}
                         </form.Field>
 
-                        <form.Field name="showWatermark">
-                            {(field) => (
-                                <div className="flex gap-5 items-center">
-                                    <label htmlFor={field.name} className={`text-sm font-medium text-grayscale-300`}>
-                                        Nango watermark
-                                    </label>
-                                    <Tooltip delayDuration={0}>
-                                        <TooltipTrigger asChild>
-                                            <div className="flex items-center">
-                                                <Switch
-                                                    id={field.name}
-                                                    name={field.name}
-                                                    checked={field.state.value}
-                                                    onCheckedChange={(checked) => field.handleChange(checked)}
-                                                    onBlur={field.handleBlur}
-                                                    disabled={!canDisableWatermark}
-                                                />
-                                            </div>
+                        {canCustomizeTheme && <ThemeColorPickers disabled={false} form={form} />}
+                        {canDisableWatermark && <WatermarkToggle disabled={false} form={form} />}
+
+                        {(canCustomizeTheme || canDisableWatermark) && (
+                            <form.Subscribe selector={(state) => [state.canSubmit, state.isDirty, state.isSubmitting]}>
+                                {([canSubmit, isDirty]) => (
+                                    <Button
+                                        type="submit"
+                                        variant="primary"
+                                        size="sm"
+                                        className="self-start"
+                                        disabled={!canSubmit || !isDirty}
+                                        loading={isUpdatingConnectUISettings}
+                                    >
+                                        Save Connect UI settings
+                                    </Button>
+                                )}
+                            </form.Subscribe>
+                        )}
+
+                        {(!canCustomizeTheme || !canDisableWatermark) && (
+                            <div className="bg-bg-subtle p-6 flex flex-col gap-6">
+                                <div className="flex text-body-medium-regular gap-2 ">
+                                    <div className="flex flex-col justify-center">
+                                        <Lock size="16" />
+                                    </div>
+                                    Advanced Customization
```

Comment:
I think this could be:
```suggestion
                                <div className="flex text-body-medium-regular gap-2 items-center">
                                    <Lock size="16" />
                                    <span>Advanced Customization</span>
```